core/src/main/java/com/amplifyframework/predictions/models/GenderAttribute.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.models;
+
+import androidx.annotation.NonNull;
+
+import java.util.Objects;
+
+/**
+ * Class that holds the gender detection results
+ * for the predictions category.
+ */
+public final class GenderAttribute {
+    private final GenderType gender;
+    private final Float score;
+
+    private GenderAttribute(
+            @NonNull GenderType gender,
+            @NonNull Float score
```

Comment:
Should this field be called `confidence`? What is the permissible range? Can you add to the docs, whatever it is? Is this in the range [0,100]?

Reply:
I was thinking of [0,1], but makes sense to make it 0-100 because thats what the AWS services output. And I agree confidence is a better name.